pdns/dnsdistdist/dnsdist-web.hh

```diff
@@ -0,0 +1,22 @@
+#pragma once
```

Comment:
Maybe add the license header ?

Reply:
Didn't we stop adding it? I'm a bit lost on that topic to be fair :-)